packages/react-charting/docs/TestPlans/GroupedVerticalBarChart/ComponentTests.md

```diff
@@ -0,0 +1,63 @@
+**Grouped Vertical Bar Chart – Component test plan**
+
+**Sub-components: Bar, Legends, Callout, Labels**
+
+1. **Bar: Bar data, Bar color, bar label**
+1. **Legends: show/hide legends, highlight the corresponding bar on legend hover**
+1. **Callout: Default/custom callout**
+1. **Labels: x-Axis labels default/rotated**
+
+|                                    **Test steps**                                     |                                              **Validation**                                               | **Tool used** |
+| :-----------------------------------------------------------------------------------: | :-------------------------------------------------------------------------------------------------------: | :-----------: |
+|                              Test 1: [Snapshot testing]                               |                                                                                                           |               |
+|                     - With only data prop, string data on x-axis.                     |                                Renders GroupedVerticalBar chart correctly                                 |    Enzyme     |
+|                   - With only data prop, numerical data on x-axis.                    |                                Renders GroupedVerticalBar chart correctly                                 |      RTL      |
```

Comment:
Grouped vertical bar chart does not support numerical data on x axis. The data you used is numeric data in string format.

Reply:
i will update the doc